starlark/unpack.go

```diff
@@ -24,8 +24,11 @@ type Unpacker interface {
 // Iterable, or user-defined implementation of Value,
 // UnpackArgs performs the appropriate type check.
 // Predeclared Go integer types uses the AsInt check.
-// If the parameter name ends with "?",
-// it and all following parameters are optional.
+//
+// If the parameter name ends with "?", it's optional.
```

Comment:
The previous text prescribes that "it and all following parameters" are optional. Is that no longer the case?

The prevailing style here avoids enclitics: "it is".